plan/constants.py

```diff
@@ -289,3 +292,29 @@ class PlanData:
 ]
 
 TRIAL_PLAN_SEATS = 1000
+
+ENTERPRISE_TIER_PLAN_NAMES = [
+    PlanName.ENTERPRISE_CLOUD_MONTHLY.value,
+    PlanName.ENTERPRISE_CLOUD_YEARLY.value,
+]
```

Comment:
Would it make sense to just include a `tier` with each plan representation?  Then we wouldn't need to keep these lists in sync and might simplify the implementation of `TierService`

Reply:
That's a good idea, my only hesitance is anticipating if a tier could ever not be strictly tied to your plan. If so, it almost feels like there's no need for a tier service altogether, and just a property of the plan service, so like 


``` python

<some>_PLAN_REPRESENTATIONS = {
    PlanName.<some>_PLAN_NAME.value: PlanData(
        ...
        tier=<some tier>,
        ...
    ),
  }
}


class PlanService():
...
def tier(self) -> TierName:
  return self.plan_data.tier
```